client/src/templates/layouts/NavigationLayout.js

```diff
@@ -622,12 +623,12 @@ const NavigationLayout = (props) => {
     </div>
   );
 
-  if (isAuthenticated) { 
-    tagManagerArgs['dataLayer'] = {userId: user.id};
+  if (isAuthenticated) {
+    tagManagerArgs["dataLayer"] = { userId: user.id };
     TagManager.initialize(tagManagerArgs);
-  } 
+  }
 
-  return <>{renderNavigationBar()}</>; 
+  return <>{renderNavigationBar()}</>;
```

Comment:
I still think that this will cause some sort of conflict once we try to merge either this to master or master to the rc branch. This code will no longer exist, and hence it will cause conflicts... Can you please just revert the formatting changes from lines 625 to 631, and just have the line 306?

Reply:
Sure, I think I can edit directly on Github and avoid our auto linting.

Reply:
Done!